From: Michael Biebl <biebl@debian.org>
Date: Sat, 9 Feb 2013 07:28:08 +0100
Subject: Remove /etc/sysconfig/firewalld support

This is a Redhatism. If users want to override how firewalld is started, they
can use the standard systemd mechanisms for that, like drop-ins.
---
 config/firewalld.service.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/config/firewalld.service.in b/config/firewalld.service.in
index b00d36c..50e32e1 100644
--- a/config/firewalld.service.in
+++ b/config/firewalld.service.in
@@ -8,8 +8,7 @@ Conflicts=iptables.service ip6tables.service ebtables.service ipset.service
 Documentation=man:firewalld(1)
 
 [Service]
-EnvironmentFile=-/etc/sysconfig/firewalld
-ExecStart=@sbindir@/firewalld --nofork --nopid $FIREWALLD_ARGS
+ExecStart=@sbindir@/firewalld --nofork --nopid
 ExecStartPost=@bindir@/firewall-cmd --state
 # don't fail ExecStartPost on RUNNING_BUT_FAILED
 SuccessExitStatus=251
